posthog/migrations/0084_org_team_user.py

```diff
@@ -0,0 +1,175 @@
+# Generated by Django 3.0.7 on 2020-09-09 10:50
+
+import uuid
+
+import django.db.models.deletion
+import django.db.models.expressions
+from django.conf import settings
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    User = apps.get_model("posthog", "User")
+    Organization = apps.get_model("posthog", "Organization")
+    OrganizationMembership = apps.get_model("posthog", "OrganizationMembership")
+    for user in User.objects.all():
+        team = user.teams_deprecated_relationship.get()
+        deterministic_derived_uuid = team.deterministic_derived_uuid
```

Comment:
This line causes running migrations locally to fail with "AttributeError: 'Team' object has no attribute 'deterministic_derived_uuid'"